ProgramowanieBot/Handlers/MessageHandler.cs

```diff
@@ -1,16 +1,27 @@
-﻿using Microsoft.Extensions.Logging;
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
 
 using NetCord;
 using NetCord.Gateway;
 
+using ProgramowanieBot.Data;
+using ProgramowanieBot.Helpers;
+
 namespace ProgramowanieBot.Handlers;
 
 internal partial class MessageHandler : BaseHandler<ConfigService.GuildThreadHandlerConfig>
 {
+    private readonly string[] _resolveKeywords;
```

Comment:
```suggestion
    private readonly IReadOnlyList<string> _resolveKeywords;
```

I am going to leave this suggestion for now while we discuss whether to cache config values ​​or not.

Reply:
I will resolve this conversation, for we decided not to cache those values.